galaxy_ng/app/access_control/statements/standalone.py

```diff
@@ -256,6 +256,22 @@
     ],
 
     'NamespaceViewSet': [
+        {
+            "action": ["ai-deny-index-list"],
```

Comment:
should we add an insights mode integration test to ensure these urls give a 404?

Reply:
Is this not supposed to work on insights as well?

If so, is there a feature flag specific to the feature, or do we gate any UI changes based on deployment mode?

Reply:
without the `ai_deny_index` feature flag the endpoints will raise 404